tools/LLVMOptimizer.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "llvm/IR/Function.h"
```

Comment:
none of these includes are needed

Reply:
I rechecked and removed most duplicate headers.